users/tests.py

```diff
@@ -1,3 +1,163 @@
-from django.test import TestCase
+from django.test import TestCase, Client
+from django.urls import reverse
+from django.contrib import auth
+
+
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APIClient
+
+from .models import User
 
 # Create your tests here.
+
+class UserPasswordTest(TestCase):
+
+    def setUp(self):
+        self.test_user = User.objects.create_user(email="test@test.com")
+
+        self.client = APIClient()
+        self.client.force_authenticate(user=self.test_user)
+    
+    def test_change_password(self):
```

Comment:
@melchiaballe per functionality create a new method. do not compile all the tests into just one method